packages/app-store/amie/DESCRIPTION.md

```diff
@@ -1,8 +1,8 @@
 ---
 items:
-  - /api/app-store/amie/1.jpg
-  - /api/app-store/amie/2.jpg
-  - /api/app-store/amie/3.jpg
+  - 1.jpg
```

Comment:
These changes are there in almost all DESCRIPTION.md files.
Relative URL was supported earlier as well, it's fixing the already created DESCRIPTION.md files that use long path.